torchci/lib/bot/drciBot.ts

```diff
@@ -35,41 +35,45 @@ export default function drciBot(app: Probot): void {
     const repo = context.payload.repository.name;
     const owner = context.payload.repository.owner.login;
     
-    console.log(pr_owner);
-    if (pr_owner in possibleUsers) {
-      const existingValidationCommentData = await getDrciComment(
-        context,
-        prNum,
+    context.log(pr_owner);
+
+    if (possibleUsers.indexOf(pr_owner) == -1) {
```

Comment:
usually it's better to use a set here instead of indexing through the array. shouldn't matter too much though.

Reply:
using `includes` instead here